mapit_gb/management/commands/mapit_UK_show_missing_codes.py

```diff
@@ -16,12 +31,13 @@ def handle_noargs(self, **options):
             gen_id=current_generation.id
         )
 
-        # These types are from
-        # http://github.com/alphagov/imminence/blob/26f6c9e5969a9e09bd24d6e2e4ebfe55dba1d997/config/routes.rb#L13
-        # except for COI which is used in Frontend:
-        # https://github.com/alphagov/frontend/blob/aed183cf3ed6a1e77cf3ec11f7dd6c238a7557cf/lib/location_identifier.rb#L4
-        used_area_types = ('EUR', 'CTY', 'DIS', 'LBO', 'LGD', 'MTD', 'UTA', 'COI')
-        used_code_types = ('ons', 'gss')  # 'ons' is the SNAC code
+        used_area_types = options['area_types'].split(',')
+        used_code_types = options['code_types'].split(',')
+
+        print 'Checking {area_types} for missing {code_types}'.format(
+            area_types=used_area_types,
+            code_types=used_code_types
+            )
```

Comment:
Minor point: the indentation of this closing bracket doesn't match the others in this file.